src/components/Product/ProductList/ProductContent/ProductRadioBox/ProductInfo/ProductInfo.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ReactComponent as RigthArrow } from '../../../../../../assets/icon/arrow_right.svg';
+import './ProductInfo.scss';
+
+const ProductInfo = ({
+  spotdata: { isNew, title, description, price, url, alt },
+  isRadioActive,
+  handleInfoMouseEnter,
+  handleInfoMouseLeave,
```

Comment:
구조분해할당은 1 depth로 하는 것이 위코드 컨벤션입니다! 
```suggestion
  spotdata,
  isRadioActive,
  handleInfoMouseEnter,
  handleInfoMouseLeave,
```

이후에 컴포넌트 내부에서 spotdata에 대해서 구조분해할당 해서 사용하는 방식으로 수정해주세요!

Reply:
네! 수정했습니다 😊